app/metrics.yaml

```diff
@@ -5445,3 +5445,35 @@ set_default_setting_experiment:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-10-01"
+
+start_on_home:
+  enter_home_screen:
+    type: counter
+    description: |
+      Know how often user lands on Homescreen upon opening the app.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/19881
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/19791
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-10-07"
```

Comment:
nit: the expiry is less than 4 months.  Shouldn't this at least be a year?

Reply:
Addressed